include/volume/math_helpers.h

```diff
@@ -0,0 +1,44 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2021 Vaibhav Thakkar
```

Comment:
please add the copyrights / names from the files you copied the functions that you pasted in this files e.g. `get_mean_variance`

Reply:
added 👍🏽